content/webapp/pages/exhibition.js

```diff
@@ -1,29 +1,38 @@
 // @flow
 import { type Context } from 'next';
 import { type UiExhibition } from '@weco/common/model/exhibitions';
-import { getExhibition } from '@weco/common/services/prismic/exhibitions';
+import {
+  getExhibition,
+  getExhibitExhibition,
+} from '@weco/common/services/prismic/exhibitions';
 import Exhibition from '../components/Exhibition/Exhibition';
 import Installation from '../components/Installation/Installation';
 
 type Props = {|
   exhibition: UiExhibition,
+  partOf: ?UiExhibition,
 |};
 
-const ExhibitionPage = ({ exhibition }: Props) => {
+const ExhibitionPage = ({ exhibition, partOf }: Props) => {
   if (exhibition.format && exhibition.format.title === 'Installation') {
-    return <Installation installation={exhibition} />;
+    return <Installation installation={exhibition} partOf={partOf} />;
   } else {
     return <Exhibition exhibition={exhibition} />;
   }
 };
 
 ExhibitionPage.getInitialProps = async (ctx: Context) => {
   const { id } = ctx.query;
-  const exhibition = await getExhibition(ctx.req, id);
+
+  const [exhibition, partOf] = await Promise.all([
```

Comment:
Could we do this on the clientside like we have elsewhere?

Reply:
If we start getting into GraphQL we could run the two queries consecutively and the resolvers should deal with most of it, just getting into the habit of having 2 async calls for 1 request, where the first request is the integral content makes me nervous.

Happy to think this logic if we think it necessary.